modcc/symge.cpp

```diff
@@ -80,7 +80,10 @@ double estimate_cost(const sym_matrix& A, pivot p) {
 // The reduction is division-free: the result will have non-zero terms
 // that are symbols that are either primitive, or defined (in the symbol
 // table) as products or differences of products of other symbols.
-void gj_reduce(sym_matrix& A, symbol_table& table) {
+// Returns a vector of vectors of symbols, partitioned by row of the matrix
+std::vector<std::vector<symge::symbol>> gj_reduce(sym_matrix& A, symbol_table& table) {
```

Comment:
Just to clarify, is it that each element of the returned vector corresponds to one row reduction operation, and contains the symbols in the reduced row?

Reply:
Correct.